### ISSUE TITLE
Training Issues and Enhancements 

### ISSUE THREAD

**Issue by ztsv-av** [ztsv-av/spellbook#11]
Classification
 - [ ] load data as batches, not whole data at once
 
Object Detection
- [ ] check if it is possible to do distributed training using Tensorflow Object Detection API

**Comment by ztsv-av** [comment]
Our code saves weights and csvs as separate files for each epoch. For weights this concept is fine, but for csvs it is a headache to check every file. Perhaps there is a more cleaner solution to this?

**Comment by ztsv-av** [comment]
For saving csv callback, check if directory is emplty. If empty => create csv, if not empty, load csv, add row, save csv with the same name.

**Comment by ztsv-av** [comment]
validation accuracy was low and unstable due to overfitting (models are too large for fashion mnist)
